src/scripts.js

```diff
@@ -134,35 +134,35 @@ $(document).ready(function () {
   });
 
   //Sleep
-  $('.hours-slept-day').text(`${sleep.returnSleepHours(date)} hours | ${sleep.returnSleepQuality(date)} quality`);
+  $('.hours-slept-day').text(`${sleep.returnsleepData(date, 'hoursSlept')} hours | ${sleep.returnsleepData(date, 'sleepQuality')} quality`);
```

Comment:
I'm not very familiar with the bitwise-or operator (except what I just <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators">googled</a>).  I'm not saying we should change it but I definitely can't explain what is happening here.